src/cdk/table/can-stick.ts

```diff
@@ -32,6 +32,7 @@ export interface CanStick {
 }
 
 /**
+ * @docs-private
```

Comment:
I think we put the `@docs-private` after the description everywhere else.

Reply:
I see, thought about creating a rule that enforces this now. Do you think it would be worth having a rule that enforces this?